samples/zigbee/cli/README.rst

```diff
@@ -0,0 +1,153 @@
+   You can use one of the following CLI interfaces:
+
+   * UART - For more details, see :ref:`putty`.
+   * RTT - For more details, see :ref:`testing_rtt`.
+
+#. Depending on whether you have access to a standalone Zigbee Coordinator:
+
+   * If you have a standalone Zigbee Coordinator, make sure it is able to accept new nodes.
+   * If you do not have a standalone Zigbee Coordinator, use one of the following solutions:
+
+     - Use a Zigbee Coordinator from the :ref:`zigbee_network_coordinator_sample` sample and flash it on a third board.
+     - Set up one of the CLIs to act as Zigbee Coordinator (see `Setting up CLI device as Zigbee Coordinator`_).
+
+#. By default, the CLI sample implements the Zigbee Router functionality.
+   If you have a coordinator nearby that is listening on channel 16, run the following command on boards programmed with the CLI sample (excluding the coordinator board, if you use a CLI board as coordinator):
+
+   .. code-block::
+
+      bdb channel 16
+
+#. Run the following command on each board programmed with the CLI sample:
+
+   .. code-block::
+
+      bdb start
+
+   The CLI nodes join the network.
+#. Check that the devices have commissioned by running the following command on each board programmed with the CLI sample:
+
+   .. code-block::
+
+      zdo short
+
+   The command returns the acquired short address of the each CLI node.
+#. Check the EUI64 addresses by running the following command on each board programmed with the CLI sample:
+
+   .. code-block::
+
+      zdo eui64
+
+#. On one of the boards programmed with the CLI sample, run the following command:
+
+   .. code-block::
+
+      zcl ping <eui64 address of the other board> 30
+
+The board pings the other board programmed with the CLI sample with 30 bytes of payload data and prints the time passed between sending a request and receiving a reply.
+
+Setting up CLI device as Zigbee Coordinator
+===========================================
+
+If you do not have access to a network coordinator (standalone or on a third board), you can have one of the development kits to act as coordinator.
+To set up the development kit:
+
+#. Build the sample, program it to your development kit, and set up the serial connection as described in `Testing`_.
+#. Set CLI device to work as a coordinator by running the following CLI command:
+
+   .. code-block::
+
+      bdb role zc
+
+-# Issue the following command on the CLI coordinator node to start a new Zigbee network:
+
+   .. code-block::
+
+      bdb start
+
+At this point, proceed with the `Testing`_ procedure steps.
+
+Dependencies
+************
+
+This sample uses the following |NCS| libraries:
+
+* Zigbee subsystem:
+
+  * :file:`zb_nrf_platform.h`
+  * :file:`zigbee_helpers.h`
+  * :file:`zb_error_handler.h`
+
+* :ref:`dk_buttons_and_leds_readme`
+
+This sample uses the following `nrfxlib`_ libraries:
+
+* :ref:`nrfxlib:zboss`
+
+In addition, it uses the following Zephyr libraries:
```

Comment:
Should the `Shell library` be mentioned here?

Reply:
You tell me -- does the sample use it?

[EDIT: Based on f2f discussion, the sample might use Zigbee Shell library, which will require documentation of its own. @sebastiandraus told me the Zephyr Shell is not used by the sample, so for now no mention of shell will be added.]

Reply:
F2F: `Zigbee shell` will be a library that will be added before CLI sample is ready. @tomchy will implement it together with @sebastiandraus .